app/templates/events/view.hbs

```diff
@@ -20,14 +20,22 @@
       <div class="twelve wide column {{unless device.isMobile 'right aligned'}}">
         {{#if device.isMobile}}
           <div class="ui icon fluid buttons">
-            <a href="{{href-to 'public' model.id}}" class="ui button">
+            <a href="{{href-to 'public' model.id}}" class="ui button" data-tooltip="Preview" data-position="top left">
               <i class="unhide icon"></i>
             </a>
-            <button class="ui button" {{action 'togglePublishState'}}>
-              <i class="{{if (eq model.state 'published') 'ban' 'check'}} icon"></i>
+              {{#if (eq model.state 'published')}}
+              <button class="ui button" {{action 'togglePublishState'}} data-tooltip="Unpublish" data-position="top left">
```

Comment:
This if can be condensed into 2 seperate inline ifs
in line 27, use 
`data-tooltip = " {{if isEventPublished 'Unpublish' 'Publish' }} "`
define a component property isEventPublished in the js file.

In line 28, use `<i class = "{{if isEventPublished 'ban' 'check' }} icon"`

Reply:
@kushthedude I don't see these changes?